blog/executor-plugins.md

```diff
@@ -0,0 +1,184 @@
+---
+slug: flux
+authors:
+  - name: "Mateusz Szostok"
+    title: "Botkube Maintainer"
+    url: https://github.com/mszostok
+    image_url: https://avatars.githubusercontent.com/u/17568639?s=250
+---
+
+# Creating the Botkube Flux Plugin for Day 2 operations
+
+Hey there, tech enthusiasts! Today we're diving into the world of GitOps with the brand-new Botkube Flux plugin.
+
+## Introduction
+
+In Botkube, we move towards GitOps by developing such extensions like Flux plugin. We all know the importance of GitOps workflow automation—it's the backbone of modern Kubernetes management. But sometimes, we encounter situations where automation is needed but hasn't been implemented yet. That's where Botkube steps in.
```

Comment:
In Botkube, we move towards GitOps by developing extensions like the new [Flux plug-in](https://botkube.io/integration/botkube-flux-kubernetes-integration).